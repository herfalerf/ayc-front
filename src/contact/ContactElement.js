import React from "react";
import { Button } from "@material-ui/core";
import { Link } from "react-router-dom";
import { makeStyles } from "@material-ui/core/styles";
import { Grid, Typography } from "@material-ui/core";

const useStyles = makeStyles((theme) => ({
  root: {
    flexGrow: 1,
  },
  contactItem: {
    backgroundImage: `url('${process.env.PUBLIC_URL}/images/contact/contactlink.png')`,
    backgroundSize: "cover",
    backgroundPosition: "right bottom",
    padding: theme.spacing(6),
    minHeight: "25vh",
  },
  button: { color: "#FFFFFF" },
  thickText: {
    fontWeight: "bold",

    color: "rgb(255,255,255)",
    padding: theme.spacing(1),
  },
  contactSubTitle: {
    color: "white",
    padding: theme.spacing(1),
  },
  contactText: {
    color: "rgb(255,255,255)",
    padding: theme.spacing(1),
  },
}));

const ContactElement = () => {
  const classes = useStyles();
  return (
    <div className={classes.root}>
      <Grid
        container
        justifyContent="center"
        alignItems="center"
        className={classes.contactItem}
      >
        <Grid item xs={12}>
          {" "}
          <Typography
            variant="subtitle1"
            className={classes.thickText}
            align="center"
          >
            OUR RESULTS SPEAK FOR THEMSELVES
          </Typography>
        </Grid>
        <Grid item xs={12}>
          {" "}
          <Typography
            variant="h4"
            className={classes.contactSubTitle}
            align="center"
          >
            Get In Touch to See How We Can Help
          </Typography>
        </Grid>
        <Grid item xs={12}>
          <Typography
            variant="subtitle1"
            align="center"
            className={classes.contactText}
          >
            Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolor
            facilis id aspernatur sit non quod fugit eaque similique consequatur
            tempora!
          </Typography>
        </Grid>
        <Grid item xs={12}>
          {" "}
          <Typography align="center" className={classes.contactText}>
            <Button
              className={classes.button}
              variant="contained"
              component={Link}
              color="primary"
              to="/contact"
            >
              Send a Message
            </Button>
          </Typography>
        </Grid>
      </Grid>
    </div>
  );
};

export default ContactElement;
