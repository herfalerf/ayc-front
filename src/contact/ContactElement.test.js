import React from "react";
import { render } from "@testing-library/react";
import ContactElement from "./ContactElement";
import { MemoryRouter } from "react-router";

it("renders without crashing", function () {
  render(
    <MemoryRouter>
      <ContactElement />
    </MemoryRouter>
  );
});

it("matches snapshot", function () {
  const { asFragment } = render(
    <MemoryRouter>
      <ContactElement />
    </MemoryRouter>
  );
  expect(asFragment()).toMatchSnapshot();
});
