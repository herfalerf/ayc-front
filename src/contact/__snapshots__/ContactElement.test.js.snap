// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="makeStyles-root-7"
  >
    <div
      class="MuiGrid-root makeStyles-contactItem-8 MuiGrid-container MuiGrid-align-items-xs-center MuiGrid-justify-content-xs-center"
    >
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
      >
         
        <h6
          class="MuiTypography-root makeStyles-thickText-10 MuiTypography-subtitle1 MuiTypography-alignCenter"
        >
          OUR RESULTS SPEAK FOR THEMSELVES
        </h6>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
      >
         
        <h4
          class="MuiTypography-root makeStyles-contactSubTitle-11 MuiTypography-h4 MuiTypography-alignCenter"
        >
          Get In Touch to See How We Can Help
        </h4>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
      >
        <h6
          class="MuiTypography-root makeStyles-contactText-12 MuiTypography-subtitle1 MuiTypography-alignCenter"
        >
          Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolor facilis id aspernatur sit non quod fugit eaque similique consequatur tempora!
        </h6>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
      >
         
        <p
          class="MuiTypography-root makeStyles-contactText-12 MuiTypography-body1 MuiTypography-alignCenter"
        >
          <a
            aria-disabled="false"
            class="MuiButtonBase-root MuiButton-root MuiButton-contained makeStyles-button-9 MuiButton-containedPrimary"
            href="/contact"
            role="button"
            tabindex="0"
          >
            <span
              class="MuiButton-label"
            >
              Send a Message
            </span>
            <span
              class="MuiTouchRipple-root"
            />
          </a>
        </p>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
